packages/ui-components/src/Icon/Icon.test.tsx

```diff
@@ -41,8 +41,11 @@ describe("Icon size prop", () => {
     wrapper.setProps({ size: "small" });
     expect(wrapper.prop("className")).toContain("size-small");
 
-    wrapper.setProps({ size: "tiny" });
-    expect(wrapper.prop("className")).toContain("size-tiny");
+    wrapper.setProps({ size: "x-large" });
```

Comment:
I appreciate the catch. I'm not sure why I added a tiny in here given that isn't a size we currently have. Would you mind re-ordering these from smallest to largest?

Reply:
What do you think about "dynamic" tests generation in this case? It might be easier to extend those test.
```typescript
const iconSizes = ["small", ... "x-large"]
iconSizes.forEach(iconSize => {
  test(`with ${iconSize} size`, () => { ... })
})
```

Reply:
@nathanstilwell , I've updated Icon tests (made more DRY)